### ISSUE TITLE
Typescript error on `storage.type`

### ISSUE THREAD

**Issue by MartialSeron** [Asjas/prisma-redis-middleware#230]
**Describe the bug**
I copied the code form README page and although the code is working well, typescript gives me an error.

Code : 
```ts
import Prisma from 'prisma';
import Redis from 'ioredis';
import { createPrismaRedisCache } from 'prisma-redis-middleware';

const redis = new Redis();

const cacheMiddleware: Prisma.Middleware = createPrismaRedisCache({
  storage: {
    type: 'redis', // error here : Type '"redis"' is not assignable to type '"memory"'.ts(2322)
    options: {
      client: redis,
      invalidation: { // error here : Type '{ referencesTTL: number; }' is not assignable to type 'boolean | undefined'.ts(2322)
        referencesTTL: 300
      },
      log: console
    }
  },
  // rest of the code
});
```

![image](https://user-images.githubusercontent.com/5698488/186376096-92e26bc6-88c2-41df-9d17-f51cebe04aab.png)

![image](https://user-images.githubusercontent.com/5698488/186376194-04c981c5-c1be-44f0-8d62-7f8baa534311.png)

**Desktop (please complete the following information):**
 - OS: VSCode (1.70.2) devcontainer on WSL2 (Debian)

**Comment by feralheart** [comment]
I have the same error.
Any update on this?

**Comment by Asjas** [comment]
No-one has opened a PR to fix it so there is no update on this.

**Comment by marvelbark2** [comment]
Yes me too, I guess the error on ioredis version (RedisMemoryOptions.client: Redis)

**Comment by marvelbark2** [comment]
This is my tmp solution

```
import type TRedis from "prisma-redis-middleware/node_modules/ioredis/built";
import Redis from "ioredis";

const redis = global.redis || new Redis(process.env.REDIS_URL || 'redis://localhost:6379');

const subRedis = redis as unknown as TRedis
const cacheMiddleware: any = global.prismaCache || createPrismaRedisCache({
    storage: { type: "redis", options: { client: subRedis, invalidation: true } },
    cacheTime: 30,
});
```

**Comment by lcdss** [comment]
This started happening to me after I updated the `ioredis` version. All `ioredis` versions  >= 5.2.6 show these errors for me. As a workaround, I just fixed the version to `5.2.5`.
